beekeeper-path-scheduler-apiary/src/main/resources/beekeeper-path-scheduler-apiary-application.yml

```diff
@@ -13,5 +13,4 @@ spring:
 properties:
   beekeeper:
     default-cleanup-delay: P3D
-  apiary:
-    cleanup-delay-property-key: beekeeper.unreferenced.data.retention.period
+    default-expiration-delay: P30D
```

Comment:
Ah so this is where the key used to come from? I think that's a good move as we're not going to change this key now.

Reply:
Yup :-)